istioctl/pkg/install/pre-check.go

```diff
@@ -67,9 +67,11 @@ func installPreCheck(istioNamespaceFlag string, restClientGetter resource.RESTCl
 	if err != nil {
 		errs = multierror.Append(errs, fmt.Errorf("failed to query the Kubernetes API Server: %v", err))
 		fmt.Fprintf(writer, fmt.Sprintf("Failed to query the Kubernetes API Server: %v.\n", err))
-	} else {
-		fmt.Fprintf(writer, "Can query the Kubernetes API Server.\n")
+		fmt.Fprintf(writer, fmt.Sprintf("Skip following checks due to: %v.\n", err))
```

Comment:
The message "Skip following checks due to:" is unclear.  The user may not know which checks are being skipped.  The `err` message will be printed twice and if that message is very long it may be difficult for the user to understand.

A better message might be 

```
fmt.Fprintf(writer, "Istio install NOT verified because the cluster is unreachable.\n")
```